introduction.md

```diff
@@ -1,3 +1,4 @@
-Hola DOJO!!!! Soy Miguel y estoy refrescando mis conocimientos de Git y GitHub
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?